### PR TITLE
Updated namespace to istio-tutorial

### PR DESCRIPTION
**Pull request by knrc**

### CODE REVIEW

File: instructions/kiali/kiali.md [hunk 1]
```diff
@@ -29,11 +29,13 @@ To open the kiali console enter the following command within a new terminal
```
Comment (brian-avery) [inline instructions/kiali/kiali.md:29]: Line 7 above, Kiali should be capitalized.

File: instructions/kiali/kiali.md [hunk 2]
```diff
@@ -49,8 +51,8 @@ You should now see the application graph showing traffic being created by our sc
 
 Within the Kiali UI select the *Distributed Tracing* option from the left hand navigation and then choose
 
-* Namespace: tutorial
-* Service: customer.tutorial
```
Comment (brian-avery) [inline instructions/kiali/kiali.md:53]: Down below, "please keep you Kiali open" should be please keep your

Comment (brian-avery) [reply instructions/kiali/kiali.md:53]: Kiali provides additional capabilties

capabilities

**Review by brian-avery — APPROVED** [review]

File: instructions/kiali/kiali.md (hunk 2 above)
Comment (knrc) [reply instructions/kiali/kiali.md:53]: @brian-avery Submit a new PR for these changes, this one is only about the namespace change.

File: instructions/kiali/kiali.md (hunk 1 above)
Comment (knrc) [reply instructions/kiali/kiali.md:29]: @brian-avery Submit a new PR for this change, this one is only about the namespace change

**Comment by thoraxe** [comment]
Should this be merged?

**Comment by knrc** [comment]
Yes please, Brian's comments concerned other parts of the already existing content and are not related to the PR.  If Brian hasn't created a PR for them then I can do so.

**Comment by brian-avery** [comment]
I've submitted a PR for the Kiali content and that's already merged. This looks good to me.

**Comment by knrc** [comment]
Thanks @brian-avery

**Pull request merged**